r/tools/nixlibs.R

```diff
@@ -333,10 +333,10 @@ build_libarrow <- function(src_dir, dst_dir) {
     env_vars <- paste(env_vars, "ARROW_JEMALLOC=OFF ARROW_PARQUET=OFF ARROW_DATASET=OFF ARROW_WITH_RE2=OFF ARROW_WITH_UTF8PROC=OFF EXTRA_CMAKE_FLAGS=-DARROW_SIMD_LEVEL=NONE")
   }
   cat("**** arrow", ifelse(quietly, "", paste("with", env_vars)), "\n")
-  status <- system(
+  status <- suppressWarnings(system(
```

Comment:
`suppressWarnings()` makes sense here, though I'm curious what warnings you've seen here in the past?

Reply:
If the command errors, it will print a warning that the command had nonzero exit. But we're handling the error with another message right after, so it's redundant/confusing.

Reply:
Ah, ah right of course